mx.sulong/testsuites.py

```diff
@@ -32,7 +32,7 @@ def compileLLVMSuite():
     print("Compiling LLVM Suite reference executables ", end='')
     tools.printProgress(tools.multicompileRefFolder(_llvmSuiteDir, _cacheDir, [tools.Tool.CLANG], ['-Iinclude'], excludes=excludes))
     print("Compiling LLVM Suite with -O0 ", end='')
-    tools.printProgress(tools.multicompileFolder(_llvmSuiteDir, _cacheDir, [tools.Tool.CLANG], ['-Iinclude'], [tools.Optimization.NONE], tools.ProgrammingLanguage.LLVMIR, excludes=excludes))
+    tools.printProgress(tools.multicompileFolder(_llvmSuiteDir, _cacheDir, [tools.Tool.CLANG], ['-Iinclude'], [tools.Optimization.NONE], tools.ProgrammingLanguage.LLVMBC, excludes=excludes))
```

Comment:
It was introduced by a previous commit, but some of the constants use the notation style `I_AM_A_CONSTANT` and others `IAMACONSTANT`. Maybe we should change these (including `LLVMIR` and `LLVMBC`) to use the `I_AM_A_CONSTANT` notation?